### PR TITLE
Add pragma to turn off library interpreter written in minimal MeTTa

### PR DESCRIPTION
**Pull request by vsbogd**
Executing `!(pragma! interpreter bare-minimal)` turns off interpreter written in minimal MeTTa and only minimal MeTTa interpreter is used.

I used "interpreter" keyword because I would like to try implementing switchable interpreter.

### CODE REVIEW

**Comment by luketpeterson** [comment]
Right now, pragma settings are global across a whole runner.  But is this something we want to change on a single-module level?

**Comment by vsbogd** [comment]
My understanding right now it is required more for experimenting with code. Generally it would be great to have ability switching interpreter on the module level.

**Review by Necr0x0Der — APPROVED** [review]

**Comment by luketpeterson** [comment]
If you want to merge this to make progress elsewhere or unblock other work, that's fine.

At some future point, I think we should decide out how settings work.  i.e. should we try an unify the settings-like mechanisms like PkgInfo (module-level), Environment (runner-level), pragmas (runner-level), REPL settings (A space created by repl app) etc.

Fundamentally some settings have different scopes and different mutability (and different mutability at different scopes!).  But maybe we can define a mechanism to capture that.

**Pull request merged**
